### PR TITLE
[KOGITO-1347] Test KogitoApp using external Infinispan and Kafka

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

https://issues.redhat.com/browse/KOGITO-1347

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/features/deploy_springboot_service.feature [hunk 1]
```diff
@@ -109,3 +111,64 @@ Feature: Deploy spring boot service
...
+        }
+      }
+      """
+    And Service "process-springboot-example" contains 1 instances of process with name "orders"
+
+    When Scale Kogito application "process-springboot-example" to 0 pods within 2 minutes
+    And Scale Kogito application "process-springboot-example" to 1 pods within 2 minutes
+
+    Then Service "process-springboot-example" contains 1 instances of process with name "orders" within 2 minutes
+    # Native application is not tested as the test is very expensive and the external Infinispan configuration shouldn't have any effect on build type
```
Comment (Sgitario) [inline test/features/deploy_springboot_service.feature:146]: I guess this comment does not apply to a springboot app.

File: test/features/deploy_springboot_service.feature [hunk 2]
```diff
@@ -109,3 +111,64 @@ Feature: Deploy spring boot service
...
+        "approver" : "john", 
+        "order" : {
+          "orderNumber" : "12345", 
+          "shipped" : false
+        }
+      }
+      """
+
+    Then GraphQL request on Data Index service returns ProcessInstances processName "orders" within 2 minutes
+    # Native application is not tested as the test is very expensive and the external Kafka configuration shouldn't have any effect on build type
```
Comment (Sgitario) [inline test/features/deploy_springboot_service.feature:174]: Same here: I guess this comment does not apply to a springboot app.

File: test/features/deploy_springboot_service.feature (hunk 1 above)
Comment (sutaakar) [reply test/features/deploy_springboot_service.feature]: Good catch, it is a copy/paste leftover

**Comment by sutaakar** [comment]
CI Job for external components successfully passed

**Pull request merged**